copy_test.go

```diff
@@ -658,6 +659,99 @@ func TestCopy_WithOptions(t *testing.T) {
 	}
 }
 
+func TestCopy_RestoreDuplicates(t *testing.T) {
```

Comment:
Need another one, like `TestCopy_SkipDuplicates` and only foo or bar will be downloaded